client/src/pages/GameListPage/GameListPage.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../scss/variables.scss";
+@import "../../scss/mixins.scss";
+@import "../../scss/index.scss";
+
+.image-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.game-image {
+  width: calc(40% - 0.5rem);
+  max-width: 15rem;
```

Comment:
See my comment above for the same specificity.

Reply:
ok i'll remove the cal method